app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -35,39 +39,87 @@
   }
 }
 
+.js-enabled .gem-c-feedback__prompt-questions {
+  display: inline-block;
```

Comment:
Why is this set? It seems to be overriding a `display: block`, removing it doesn't seem to make any difference?